packages/smooth_app/lib/pages/product/product_image_viewer.dart

```diff
@@ -0,0 +1,134 @@
+// ignore_for_file: cast_nullable_to_non_nullable
```

Comment:
Just don't!

Reply:
In my defense, I can tell you that was VSCode putting it there for me (i just ignored the warning via GUI, didn't know it wasn't temporary!)